stdlib/2and3/binascii.pyi

```diff
@@ -0,0 +1,45 @@
+# Stubs for binascii
+
+# Based on http://docs.python.org/3.2/library/binascii.html
+
+import sys
+from typing import Union, Text
+
+
+if sys.version_info < (3,):
+    # Python 2 accepts unicode ascii pretty much everywhere.
+    _Bytes = Union[bytes, Text]
+    _Ascii = Union[bytes, Text]
+elif sys.version_info <= (3, 2):
```

Comment:
should be `< (3, 3)`. But we're probably going to drop 3.2 support anyway (#1145).